dataland-frontend/tests/e2e/fixtures/frameworks/sfdr/SfdrPreparedFixtures.ts

```diff
@@ -198,3 +198,24 @@ function generateFixtureWithIncompleteReferencedReport(input: FixtureData<SfdrDa
   };
   return input;
 }
+
+/**
+ * Generates an SFDR dataset with three data points that only contain a value, a quality and a comment.
+ * @param input Fixture data to be manipulated
+ * @returns the dataset
+ */
+function generateFixtureWithDifferentExtendedDatapointCases(input: FixtureData<SfdrData>): FixtureData<SfdrData> {
+  input.companyInformation.companyName = 'TestForDataPointDisplayLogic';
+  if (input.t.environmental?.greenhouseGasEmissions?.scope1GhgEmissionsInTonnes) {
```

Comment:
general info for me: why are these if-cases necessary? I mean, if these if-statements returned false, you wouldn't create an appropriate fixture right?

Reply:
Ja ich verstehe dass das komisch wirkt.
Grund ist, dass typescript dir nur erlaubt, Felder zu setten, wenn auch sichergestellt ist, dass das Objekt existiert.
Wenn ich das if wegnehme, wird typescript meckern.

Es gibt jetzt einige Alternativmöglichkeiten (Objekt literally bauen, Code schreiben à-la "wenns-Objekt-nicht-gibt-erstelle-es" etc.).
Aber die kommen alle nicht mit so wenigen Zeilen aus wie das hier 😄 

Da diese maniuplator-Funktion auf ein "volles" datset angewendet wird, sollte es auch immer klappen.